src/main/java/uk/gov/ons/census/casesvc/service/RmCaseUpdatedService.java

```diff
@@ -53,7 +55,13 @@ public void processMessage(ResponseManagementEvent rme, OffsetDateTime messageTi
     if (shouldSendCaseToField(updatedCase, rme.getEvent().getChannel())) {
       eventMetadata = new Metadata();
       eventMetadata.setCauseEventType(rme.getEvent().getType());
-      eventMetadata.setFieldDecision(ActionInstructionType.CREATE);
+
+      // Only send a CREATE on a case that field doesn't already know about
```

Comment:
For me, this "here be dragons" comment is not big and bold enough. Anybody reading our code will have absolutely no idea why the "OA being present" means "we should send an update" - we need to explain very clearly here, with big disclaimers, that we have been forced to do this, and we really don't want to be doing it like this - it's an ugly hack that we were forced to do; that it should be refactored away and done properly, when time allows; tech debt

Reply:
^ I think that should be the comment 😂

Reply:
I've updated the comment to add a disclaimer about us being forced to do it, should be refactored etc.  'Here be dragons' went right over my head 😂

Reply:
I'd still like a little more explanation about what the heck this is.

Suggested wording:
```
HERE BE DRAGONS!!
We don't want to send an UPDATE for cases which FWMT-G already know about, but the only way we've got of **guessing** that is by looking to see if an OA is on the case or not. We imagine that OA would only be set on cases which FWMT-G know about, so we have been forced to use it as an ugly kludge, because of time pressure. TODO: This should be refactored/done properly. It's tech debt.
```

Reply:
Done 👍